### PR TITLE
remove binaries from source tree

### PR DESCRIPTION
**Pull request by despair86**
- fix a bug in network auto config on windows
- bump to v0.5.1
- download windows bits at runtime
- ~~download a known good lokinet-bootstrap at installer build time as it's non-trivial to build for windows (requires TLS)~~
- ~~refactor lokinet-bootstrap for windows, more robust http response parsing~~
- ryan's macOS install bits
see #816

### CODE REVIEW

File: contrib/lokinet-bootstrap-winnt/bootstrap.c [hunk 1]
```diff
@@ -2452,22 +2453,136 @@ const void *b;
 size_t bn;
 size_t s;
 {
-  char *p = malloc(s * (an + bn));
```
Comment (majestrate) [inline contrib/lokinet-bootstrap-winnt/bootstrap.c]: please use 2 spaces not tabs kthnx

File: win32-setup/Makefile [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+
+all: regdbhelper.dll lokinet-bootstrap.exe
+
+default: all
+
+regdbhelper.dll:
+	i686-w64-mingw32-gcc regdb_helper.c -o $@ -shared -Os -s
+
+lokinet-bootstrap.exe:
+	wget https://snowlight.net/loki/win32-dist/lokinet-bootstrap.exe
```
Comment (majestrate) [inline win32-setup/Makefile]: this target should build the executable not download it.

**Comment by majestrate** [comment]
honestly it should just require libcurl

**Comment by despair86** [comment]
ok custom code replaced with a JIT build of libcurl+mbedtls to match all other platforms

**Comment by despair86** [comment]
`LICENSE` changed to have windows line endings to placate UI about box

**Comment by despair86** [comment]
ok made crlf conversion run at installer build time instead

**Pull request merged**